README.md

```diff
@@ -23,10 +23,10 @@ To build the latest development sources:
 
 (To build from git, you will need SWIG.)
 
-Once it is installed, `#include <nlopt.h>` in your C/C++ programs and
+Once it is installed, `#include "nlopt.h"` in your C/C++ programs and
```

Comment:
nlopt.h is installed into system include paths.   You normally use `"...."` only for include files within your own project, not for external libraries like nlopt.